docs/networking/harvester-network.md

```diff
@@ -64,8 +65,13 @@ Optional: Users can customize each node's VLAN network via the **HOST > Network*
 A new VLAN network can be created via the **Advanced > Networks** page and clicking the **Create** button.
 
  - Specify the name and VLAN ID that you want to create for the VLAN network.
-
-  ![create-vlan-network.png](./assets/create-network.png)
+   ![create-vlan-network.png](assets/create-network.png)
+  
+ - Configure the route so that users can connect the IPv4 address in the VLAN network from the hosts. The CIDR and gateway of the VLAN network are necessary parameters of the route. There are two methods to get them.
+   - auto(DHCP) mode: Harvester network controller will get the CIDR and gateway through DHCP protocol if there is a DHCP server. Moreover, users can specify the DHCP server address.
```

Comment:
```suggestion
   - auto(DHCP) mode: the Harvester network controller will get the CIDR and gateway values from the DHCP server using the DHCP protocol. You need to specify the DHCP server address.
```

Reply:
It's optional to specify the DHCP server address.

Reply:
rephrased based on the feedback